db/schema.rb

```diff
@@ -114,14 +114,43 @@
     t.string "ghg"
     t.string "estimated_emission_reduction"
     t.string "cobenefits"
-    t.boolean "bur1"
+    t.boolean "quantified_effect"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.index ["actor"], name: "index_mitigation_actions_on_actor"
     t.index ["ghg"], name: "index_mitigation_actions_on_ghg"
     t.index ["status"], name: "index_mitigation_actions_on_status"
   end
 
+  create_table "mitigation_effects", force: :cascade do |t|
```

Comment:
I think the effects part is not merged yet, maybe makes sense to keep away from schema here

Reply:
Yes, I needed to regenerate the schema. I just did another commit with it.